crates/matrix-sdk-appservice/src/error.rs

```diff
@@ -108,15 +114,3 @@ impl From<warp::Rejection> for Error {
         Self::WarpRejection(format!("{:?}", rejection))
     }
 }
-
-impl From<matrix_sdk::HttpError> for Error {
-    fn from(e: matrix_sdk::HttpError) -> Self {
-        matrix_sdk::Error::from(e).into()
-    }
-}
-
-impl From<matrix_sdk::StoreError> for Error {
-    fn from(e: matrix_sdk::StoreError) -> Self {
-        matrix_sdk::Error::from(e).into()
-    }
-}
```

Comment:
Why introduce new variants instead of having them deduplicated between the appservice and SDK error types?